src/main/java/bio/terra/app/logging/LoggerInterceptor.java

```diff
@@ -39,6 +46,20 @@ public void afterCompletion(
 
     // skip logging the status endpoint
     if (!url.endsWith("/status")) {
+
+      Map<String, Object> stackDriverPayload = new HashMap<>();
+      if ("post".equalsIgnoreCase(method)) {
+        String postBody =
+            new String(
```

Comment:
I don't think you need new String() here, collect(Collectors.joining()) should return you the String object, the "" is unnecessary too.

Reply:
Good catch on the collect. The `new String()` is wrapping the ByteArray, which now that I look at again, means I don't need to collect at all...